### ISSUE TITLE
Feature Request: 12 Hour Format

### ISSUE THREAD

**Issue by DeMoN100** [simplex-chat/simplex-chat#1341]
May we please have a 12 hour AM/PM format for timestamps? I really dislike being compelled against my will to use 24 hour format.

**Comment by epoberezkin** [comment]
is it on android?

**Comment by DeMoN100** [comment]
Yes the 24h timestamp there is no 12hour timestamp format on android

**Comment by quite** [comment]
I'm on simplex 5.6 (102) and Android 13 with 24-hour in the settings. Still I only see 12h-format with AM/PM (very annoying for me).

**Comment by quite** [comment]
> hour format depends on chosen language, if your language uses 24h format, it will show that, otherwise 12h.

That's really not very helpful advice. I know that you can change language within the Simplex app on Android, but I want it to be English, and I want to have 24h time format (and also ISO dates etc)

Now days, setting 12h/24h time format separate from the interface language is possible in Android (and many other operating systems), and Simplix ought to honour this user preference.

**Comment by cobber** [comment]
I would like to second the suggestion for explicit date format selection, just assuming date/time formats based on language selection is not enough.

English speakers don't agree on a single date format, and commonly use e.g. "7/3" to mean "7th of March" or "3rd of July".

I never know if an "x/y" date was formatted using _my_ selected locale (day/month), or whether the app developer was just lazy and assumed "month/day". The only way to tell is if one of the numbers is >12.

Also, as a nerd, I would much prefer the ISO 8601 format, e.g. "2024-03-07 16:45".

The key point being "its a personal preference" and the user should be able to explicitly choose.

**Comment by zplus11** [comment]
Hi, is there any progress on this? For making this an option to choose between the two? Great app btw 👍 @Narasimha-sc
